File: C99/1100-1199/1130-1139/1134.c
#include <stdio.h>

int main()
{
    int num, a = 0, g = 0, d = 0, conf = 0;
    while (conf == 0)
    {
        scanf("%d", &num);
        if (num == 1)
        {
            a++;
        }
        else if (num == 2)
        {
            g++;
        }
        else if (num == 3)
        {
            d++;
        }
        else if (num == 4)
        {
            conf++;
        }
    }
    printf("MUITO OBRIGADO\n");
    printf("Alcool: %d\n", a);
    printf("Gasolina: %d\n", g);
    printf("Diesel: %d\n", d);
    return 0;
}

File: python/1100-1199/1130-1139/1134.py
alcool = gas = diesel = 0

while True:
    entry = int(input())
    if entry == 1:
        alcool += 1
    if entry == 2:
        gas += 1
    if entry == 3:
        diesel += 1
    if entry == 4:
        break
print("MUITO OBRIGADO")
print(f"Alcool: {alcool}")
print(f"Gasolina: {gas}")
print(f"Diesel: {diesel}")